releases/v20.2.0.md

```diff
@@ -0,0 +1,126 @@
+
+Version | Feature | Description
+--------|---------|------------
+Core | **Backup/Restore in Core Version** | The core version of CockroachDB now lets you perform full cluster backups, all restore options, as well as bulk exports. Incremental backups, locality-aware backups, and other advanced backup functionality continue to require an enterprise license.
+Enterprise | **KMS Support for Encrypted Backups** | You can now use [AWS Key Management Service (KMS)](../v20.2/take-and-restore-encrypted-backups.html#use-aws-key-management-service) to encrypt the files that full or incremental backups generate.
+Enterprise | **Backup Scheduling** | You can now [create schedules for CockroachDB backups](../v20.2/manage-a-backup-schedule#create-a-new-backup-schedule), as well as [view, pause, resume, and drop backup schedules](../v20.2/manage-a-backup-schedule#view-and-control-the-backup-schedule). Once a scheduled backup is created, you can use [`SHOW BACKUP`](../v20.2/manage-a-backup-schedule#view-scheduled-backup-details) to view the backup details. Note that backup scheduling for full backups is included in the core version of CockroachDB.  
+Core | **Import with User-Scoped Storage** | In addition to supporting bulk imports from cloud storage, CockroachDB now lets you [upload CSV files from your local machine to user-scoped file storage](../v20.2/use-userfile-for-bulk-operations.html) in your cluster. Once uploaded, a userfile can be reference by the `IMPORT` command to import data into a table. Userfiles can also be listed and deleted via CLI commands.
+Core | **Import with Default Expression** |
+
+### Deployment and Operations
+
+Version | Feature | Description
+--------|---------|------------
+N/A | Kubernetes Operator | NEED RYAN'S HELP TO SUMMARIZE THIS AND LINK TO THE RIGHT PLACE(S).
+Core | **Log Redaction** | When gathering log files via the [`cockroach debug zip`](../v20.2/cockroach-debug-zip.html) or [`cockroach debug merge-logs`](../v20.2/cockroach-debug-merge-logs.html) command, you can use the new `--redact-logs` flag to redact sensitive PII data. Note that this flag removes sensitive information only from the log files; other items collected by the `debug zip` command may still contain sensitive information.
+Core | **Certificate Revocation with OCSP** | CockroachDB now supports [certificate revocation](../v20.2/create-security-certificates-custom-ca.html#certificate-revocation-with-OCSP) for custom CA certificate setups running an OCSP server.
+Enterprise | **SSO in the Admin UI** | The CockroachDB Admin UI now supports [single sign-on (SSO)](../v20.2/sso.html) via OpenID Connect (OIDC), an authentication layer built on top of OAuth 2.0. When SSO is configured and enabled, the [Admin UI login page](admin-ui-overview.html#admin-ui-access) displays an OAuth login button in addition to the password access option.
+Core | **Permission-Based Object Ownership** | NEED AMRUTA'S HELP TO SUMMARIZE THIS AND LINK TO THE RIGHT PLACE(S).
```

Comment:
@Amruta-Ranade or @ericharmeling, I need your help drafting this description and finding the right link(s).